provider/identityprovider/ldap.go

```diff
@@ -74,11 +73,7 @@ var ldapAttrSchema = map[string]schema.Attribute{
 		Description: "The list of attributes whose values should be used as the email address.",
 		ElementType: types.StringType,
 		Optional:    true,
-		Validators: []validator.List{
-			listvalidator.ValueStringsAre(
-				stringvalidator.RegexMatches(common.EmailRegexp, "Invalid email"),
-			),
-		},
+		Computed:    true,
```

Comment:
I'm not sure I understood - why did you change it to be computed?

Reply:
Honestly not 100% sure what computed does here, but I figured it would make sense to be consistent with each of the other LDAP fields which have computed set to true


For example:
```
	"id": schema.ListAttribute{
		Description: "The list of attributes whose values should be used as the user ID. (default ['dn'])",
		ElementType: types.StringType,
		Optional:    true,
		Computed:    true,
	},
	"name": schema.ListAttribute{
		Description: "The list of attributes whose values should be used as the display name. (default ['cn'])",
		ElementType: types.StringType,
		Optional:    true,
		Computed:    true,
	},
```

Reply:
now I am wondering if they all should not be computed... @jland-redhat Did you test this manually? With an actual LDAP service?